hera.rb

```diff
@@ -0,0 +1,69 @@
+class Hera < Formula
+  desc "Hera: Ewasm virtual machine conforming to the EVMC API"
+  homepage "https://github.com/ewasm/hera"
+
+  stable do
+    url "https://github.com/ewasm/hera/archive/v0.2.4.tar.gz"
```

Comment:
Couple things: brew formulas aren't really made for the pattern of "download stable binary, install might check out the source instead" as the mixing and matching of "move these files here" and "run make install" isn't a clean pattern here.

Because of that, I decided to still go with the source download.

Reply:
Also, for the record, we can build bottles for this stuff and host them on [Bintray](https://bintray.com/), as they're free for Open Source.